Public/Src/Pips/Dll/Operations/Process.cs

```diff
@@ -220,6 +223,14 @@ public sealed partial class Process : Pip
         [PipCaching(FingerprintingRole = FingerprintingRole.Semantic)]
         public ReadOnlyArray<AbsolutePath> UntrackedScopes { get; }
 
+        /// <summary>
+        /// File Path of which the Source Change Affected Inputs are written into.
+        /// </summary>
+        /// <remarks>
+        /// </remarks>
+        [PipCaching(FingerprintingRole = FingerprintingRole.None)]

```

Comment:
>FingerprintingRole.None [](start = 41, length = 23)

Why is it not `FingerprintingRole.Semantic`? I think it should be `Semantic`, right?

Reply:
This is the global affected list of the whole build. It represent the state of the build by the time this pip execute. It shouldn't be in the fingerprint.